packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -520,6 +520,9 @@ class SliverGeometry extends Diagnosticable {
   /// If this is non-zero after [RenderSliver.performLayout] returns, the scroll
   /// offset will be adjusted by the parent and then the entire layout of the
   /// parent will be rerun.
+  ///
+  /// If a parent does not consume this value by making layout corrections,
```

Comment:
I think you're referring to the parent (not an ancestor) as in the previous paragraph. So "the parent" instead of "a parent".

By "must be propagated up it its own" I think you mean: must be propagated up to its own.

This comment implies some things I don't understand:
- What does it mean for the parent to consume the scrollOffsetCorrection
- What does "its own [SliverGeometry]" refer to?

If this stuff wouldn't be obvious to someone reading more of the sliver docs, it would be helpful to explain things a little more here.

Reply:
Reworded this section.